### PR TITLE
Chore/rename engineer guild

### PR DESCRIPTION
**Pull request by miiila**
task: Rename Engineer's guild to "Architect's guild", plus rename associated walker to "Architect"

Doing precisely what the task says **without walker change**. I am not sure about common practices, so I'll be happy to address any comments, remove commits or doing whatever is needed. I will also add walker change, however I need a bit more time for that one.

Therefore I did a change in two commits, if renaming the assets wouldn't be the right thing.

### CODE REVIEW

**Comment by MSTVD** [comment]
I guess I am not the only one who wants to see this happen ;)

**Comment by crudelios** [comment]
Is this tested? Can I merge?

**Comment by miiila** [comment]
@crudelios I tested locally in my own game. Not sure if there is any external testing usually done :-)

Up to you if you want to merge now and I will raise walker update in separate PR or if you want to wait for walker change here.

**Comment by crudelios** [comment]
@miiila What are you planning to do with the walkers? Just to know if it's worth waiting to merge!

**Comment by miiila** [comment]
@crudelios I am planning to rename walker from `engineer` to `architect`, but I wanna verify I rename the right `engineer`. I assume I can do it by the EOW.

**Comment by crudelios** [comment]
Okay, this can wait, no problem!

**Comment by miiila** [comment]
@crudelios Ready for merge/review!

File: src/translation/portuguese.c [hunk 1]
```diff
@@ -109,8 +109,8 @@ static translation_string all_strings[] = {
     { TR_HOTKEY_EDIT_TITLE, "Pressiona nova tecla de atalho" },
     { TR_BUILDING_ROADBLOCK, "Bloqueio de Estrada" },
     { TR_BUILDING_ROADBLOCK_DESC, "Bloqueio de estrada - impede cidadãos ociosos." },
-    { TR_BUILDING_ENGINEER_GUILD, "Guilda de Engenharia" },
-    { TR_BUILDING_ENGINEER_GUILD_DESC, "Os engenheiros trabalham incansavelmente, construindo monumentos pela glória de Roma." },
+    { TR_BUILDING_ARCHITECT_GUILD, "Guilda de Engenharia" },
```
Comment (crudelios) [inline src/translation/portuguese.c:112]:
```suggestion
    { TR_BUILDING_ARCHITECT_GUILD, "Guilda de Arquitetos" },
```

File: src/translation/portuguese.c [hunk 2]
```diff
@@ -109,8 +109,8 @@ static translation_string all_strings[] = {
...
+    { TR_BUILDING_ARCHITECT_GUILD_DESC, "Os engenheiros trabalham incansavelmente, construindo monumentos pela glória de Roma." },
```
Comment (crudelios) [inline src/translation/portuguese.c:113]:
```suggestion
    { TR_BUILDING_ARCHITECT_GUILD_DESC, "Os arquitetos trabalham incansavelmente, construindo monumentos pela glória de Roma." },
```

**Comment by crudelios** [comment]
Apart from the suggested translation changes to portuguese, it looks good to go!

**Pull request merged**